modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureDM.testcase

```diff
@@ -38,118 +44,110 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-133993. Verify if its possible collect Digital Signature By button"
+	@description = "LPS-133993. Verify if its possible collect Digital Signature By button"
 	@priority = 5
 	test CanCollectDigitalSignatureByButton {
-		property solutions.acceptance = "true";
-
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
+		task ("Given that user Test creates a document") {
+			property solutions.acceptance = "true";
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMDocument.addCP(
+				dmDocumentDescription = "DM Document Description",
+				dmDocumentFile = "Alice's Adventures in Wonderland.pdf",
+				dmDocumentTitle = "DM Document PDF");
 
-		DMDocument.addCP(
-			dmDocumentDescription = "DM Document Description",
-			dmDocumentFile = "Alice's Adventures in Wonderland.pdf",
-			dmDocumentTitle = "DM Document PDF");
-
-		var dmDocumentTitle = "DM Document PDF";
+			var dmDocumentTitle = "DM Document PDF";
+		}
 
-		Click(
-			dmDocumentTitle = ${dmDocumentTitle},
-			locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_SELECT_DOCUMENT");
+		task ("When the user goes to digital signature") {
+			DigitalSignature.documentDigitalSignature(documentlist = "DM Document PDF");
+		}
 
-		AssertElementPresent(locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_COLLECT_BUTTON");
+		task ("Then confirm that you are on the page and delete the file") {
+			DigitalSignature.assertDigitalSignaturePage(assertPage = "New Digital Signature Envelope");
```

Comment:
Just a typo here: `"The the user can confirm that he are on Digital Signature page"`